Cargo.toml

```diff
@@ -39,15 +39,15 @@ log = "0.4"
 microservice-containers = { path = "microservice-containers" }
 microservice-derive = { path = "microservice-derive" }
 num_cpus = "1"
-paperclip = { version = "0.6", features = ["actix4", "rust_decimal", "chrono", "swagger-ui"], optional = true }
+paperclip = { git = "https://github.com/sfisol/paperclip", branch = "actix-files", features = ["actix4", "rust_decimal", "chrono", "swagger-ui"], optional = true }
 r2d2-diesel = { version = "1.0", optional = true }
 r2d2 = { version = "0.8", optional = true }
 serde = "1"
 serde_json = { version = "1" }
 weighted-rs = { version = "0.1", optional = true }
 colored = "2.0"
 
-[patch.crates-io]
-paperclip = { git = "https://github.com/paperclip-rs/paperclip" }
-# paperclip = { git = "https://github.com/sfisol/paperclip", branch = "mountable-actix-files" }
+# [patch.crates-io]
+# paperclip = { git = "https://github.com/paperclip-rs/paperclip" }
+# paperclip = { git = "https://github.com/sfisol/paperclip", branch = "actix-files" }
 # paperclip = { path = "../../rust/paperclip" }
```

Comment:
**just a nit:** consider removing it as it seems to serve no purpose but I'm fine with keeping it if you want